diff -uNr logfmon-0.7.ORIG/make-linux.sh logfmon-0.7/make-linux.sh
--- logfmon-0.7.ORIG/make-linux.sh	2005-02-11 23:20:14.000000000 +0100
+++ logfmon-0.7/make-linux.sh	2005-09-14 13:57:44.000000000 +0200
@@ -10,7 +10,7 @@
 CC=gcc
 CFLAGS="-I- -I. -I/usr/local/include $CFLAGS -D_LARGEFILE_SOURCE \
         -Wall -W -Wmissing-prototypes -Wmissing-declarations \
-        -Wshadow -Wpointer-arith -Wcast-qual -Wsign-compare"
+	-Wshadow -Wpointer-arith -Wcast-qual -Wsign-compare -D_GNU_SOURCE $@"
 LDFLAGS="-L/usr/local/lib"
 LIBS="-lm -lpthread"
 
