--- examples/logfmon-openbsd.conf	2004-11-28 16:07:35.000000000 +0100
+++ examples/logfmon-openbsd.conf	2004-12-09 00:36:37.431601656 +0100
@@ -1,12 +1,12 @@
 set mailtime 1m
 
 # Mail command. This may be /bin/mail on some systems.
-set mailcmd "/usr/bin/mail -s \"`hostname` `date` log report\" root"
+set mailcmd "/bin/mail -s \"`hostname` `date` log report\" root"
 
 file "/var/log/messages" tag messages
 file "/var/log/maillog" tag mail
 file "/var/log/secure" tag secure
-file "/var/log/authlog" tag auth
+#file "/var/log/authlog" tag auth
 
 match in * "last message repeated [0-9] times" ignore
 
